docs/Information_monitoring.rst

```diff
@@ -7,6 +7,9 @@ This information is exposed in two ways:
 
 * For every node, there is a web interface from Netdata itself at ``<node>.server.mila.quebec:19999``.
   This is accessible only when using the Mila wifi or through SSH tunnelling.
+
+  * SSH tunnelling: on your local machine, run ``ssh -L19999:<node>.server.mila.quebec:19999 -p 2222 login.server.mila.quebec``,
+    then open http://localhost:19999 in your browser.
```

Comment:
Should we replace the port 19999 by `<PORT>`? I fear that many will end up using the 19999 port on the same node which could potentially bring some issue

Reply:
It has to be port `19999` on the compute node because that is where Netdata is serving the data. There's only going to be an issue if a user wants to monitor two different nodes on the same machine, in which case they'd have to use different local ports, but I don't know if it's worth mentioning, it's not super hard to figure out.

Reply:
I think it's clear enough. If one try to open a connection to a different port with the same local port, ssh will display:
```
bind [127.0.0.1]:19999: Address already in use
channel_setup_fwd_listener_tcpip: cannot listen to port: 19999
Could not request local forwarding.
```